packages/blade/src/components/Counter/Counter.tsx

```diff
@@ -47,24 +54,37 @@ type ColorProps = {
   backgroundColor: StyledCounterProps['backgroundColor'];
 };
 
+const isFeedbackVariant = (variant: string): variant is Feedback => {
+  const feedbackVariants = ['information', 'negative', 'neutral', 'notice', 'positive'];
+  return feedbackVariants.includes(variant);
+};
+
 const getColorProps = ({
-  intent = 'neutral',
+  variant = 'neutral',
   contrast = 'low',
 }: {
-  intent: NonNullable<CounterProps['intent']>;
+  variant: NonNullable<CounterProps['variant']>;
   contrast: NonNullable<CounterProps['contrast']>;
 }): ColorProps => {
   const props: ColorProps = {
-    textColor: `feedback.text.${intent}.${contrast}Contrast`,
-    backgroundColor: `feedback.background.${intent}.${contrast}Contrast`,
+    textColor: 'feedback.text.neutral.lowContrast',
```

Comment:
why hardcode it? you're already setting the default value on the args above right?

Reply:
Yes but variant can contain `blue` and there is no `feedback.text.blue.lowContrast`, that check happens below. ColorProps type is required just to make sure it never gets undefined in the future so assigning a value here is needed.  Other option is doing it all inline which looks too complicated for no reason. Just had to pick one approach. 

```
const props: ColorProps = {
    textColor: isFeedback
      ? `feedback.text.${variant}.${contrast}Contrast`
      : `badge.text.${variant}.${contrast}Contrast`,
    backgroundColor: isFeedback
      ? `feedback.background.${variant}.${contrast}Contrast`
      : `badge.background.${variant}.${contrast}Contrast`,
  };
```